include/cantera/base/ct_defs.h

```diff
@@ -80,6 +80,9 @@ const double OneAtm = 1.01325e5;
 //! One bar [Pa]
 const double OneBar = 1.0E5;
 
+//! Unified atomic mass unit (Dalton) [kg]
+const double Dalton = 1.66053906660e-27;
+
```

Comment:
This doesn't appear to be used anywhere, do you need to add it? Does specifying this overspecify any of the other constants? In other words, can this constant be calculated from a combination of the other ones we've defined?

Reply:
This is used in line 54 of WeaklyIonizedGas.cpp to calculate the mass ratio of electron to a molecule. I will check if there is any replacement.

Reply:
Thanks, I was tricked because the "large diff" didn't load fully, my mistake! Don't spend too long on it, I just want to make a sanity check that we aren't overconstraining the system. I suspect we're not, but just to double check.

By the way, on those lines, you can move most of the calculation (at least `mass_ratio`) outside the inner loop, since it doesn't change on each iteration of `i`, only `k`.

Reply:
BTW I found out that I should use Avogadro instead of Dalton. I was confused about the definition of molecular weight.

Reply:
Yeah, your answers will be off by ~60 orders of magnitude if you flip them 😄